tests/test_dump_file.py

```diff
@@ -187,3 +187,71 @@ def test_copy_from(snap, tmp_path):
     assert numpy.all(read_snap.molecule == [0, 1, 2])
     assert read_snap.has_charge()
     assert numpy.allclose(read_snap.charge, [0, 1, -1])
+
+
+def test_faulty_dump_schema(snap, tmp_path):
+    # test image schema
+    f = open(tmp_path / "atoms.lammpstrj", "w")
+    f.write(
+        """
+    ITEM: TIMESTEP
+    0
+    ITEM: NUMBER OF ATOMS
+    1
+    ITEM: BOX BOUNDS pp pp pp
+    -7.0 7.0
+    -7.0 7.0
+    -7.0 7.0
+    ITEM: ATOMS id type ix iy
+    1 1 0 0 """
```

Comment:
I think this will allow the strings to be joined together and also use f-strings to format the schema.
```suggestion
    -7.0 7.0
    """
    f"ITEM: ATOMS id type {schema}\n"
    f"1 1 {schema_zeros}\n"""
```